tests/test_processors.py

```diff
@@ -151,3 +156,19 @@ def brand(self):
     )
     page = MyProductPage(response=response)
     assert page.brand == "foo"
+
+
+@pytest.mark.parametrize(
+    "input_value,expected_value",
+    [
+        ("", None),
+        ("1", "1.00"),
+        ("1.2", "1.20"),
+        ("1.23", "1.23"),
+        # Fails due to . being considered a thousands separator in this case.
+        # ("1.234", "1.234"),
```

Comment:
I wonder if we should expose [decimal handling override](https://github.com/scrapinghub/price-parser#decimal-separator), maybe the way we allow overriding the currency (class variable).